src/plugins/intel_gna/src/gna_graph_compiler.hpp

```diff
@@ -34,6 +34,7 @@ class GNAGraphCompiler {
     std::shared_ptr<GNAPluginNS::gna_memory_type> gnamem;
     std::shared_ptr<GNAPluginNS::GNAFlags> gnaFlags;
     std::shared_ptr<GNAPluginNS::GnaInputs> inputs_ptr_;
+    bool use_only_cnn2D_iface;
```

Comment:
And I think we might consider passing plugin configuration to backend, instead of creating new class fields.

Reply:
done as suggested